user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_LDAP_settings.md

```diff
@@ -25,8 +25,9 @@ To do so:
 
    - **Host**: The name or IP of the LDAP server.
    - **Port**: The port to connect to the LDAP server.
-   - **Authentication type**: The authentication method to access the LDAP server. The following authentication types are available: *Anonymous*, *Max*, *Member System*, *Password*, *SASL*, and *Simple*.
+   - **Authentication type**: The authentication method to access the LDAP server. The following authentication types are available: *Anonymous* and *Simple*.
    - **Naming context**: A suffix that identifies the top entry of the LDAP hierarchy.
+   - **Non domain LDAP**: An LDAP server that is not Windows Active Directory. This will be used to determine if DataMiner can subscribe to change notifications of the LDAP server, a feature that is only supported by Active Directory. DataMiner will only sync changes with a non-domain LDAP server through the *ReloadLDAP.js* script, which runs on an hourly basis through the *Skyline DataMiner LDAP Resync* scheduled task in Windows.
```

Comment:
From which DataMiner version onwards is this setting available? If it is a more recent version than 10.1.0 [CU11]/10.2.2, could you add the version information? Otherwise, this will be confusing for users who do not have the latest DataMiner version.

Also, could you confirm that the spelling of this setting is correct? I'd expect it to be "Non-domain LDAP".

Reply:
Confirmed to be 10.3.6/10.4.0 (RN 35782)